### PR TITLE
remove tasks from redis after they complete

### PR DESCRIPTION
**Pull request by terrazoon**
## Description

To keep Redis from filling up, eject tasks as soon as they complete.

## Security Considerations

N/A

### CODE REVIEW

**Review by xlorepdarkhelm — APPROVED** [review]

File: app/config.py [hunk 1]
```diff
@@ -168,6 +168,9 @@ class Config(object):
 
     CELERY = {
         "worker_max_tasks_per_child": 500,
+        "task_ignore_result": True,
+        "result_expires": 0,
+        "result_persistent": False,
```
Comment (ccostino) [inline app/config.py:173]: A quick check on all of these settings:

* [`task_ignore_result`](https://docs.celeryq.dev/en/stable/userguide/configuration.html#task-ignore-result) seems like a good one to have here, but the note in the docs talk about another setting, [`task-store-errors-even-if-ignored`](https://docs.celeryq.dev/en/stable/userguide/configuration.html#task-store-errors-even-if-ignored), that be worth thinking about from an error handling and recovery perspective.

* [`result_expires`](https://docs.celeryq.dev/en/stable/userguide/configuration.html#result-expires) defaults to `1 day`; a value of `0` or `None` means things never expire - are we sure we want that?  There are some additional notes [in this Stackoverflow post](https://stackoverflow.com/questions/63081704/clearing-out-redis-from-celery-tasks-that-has-finished) that point to this being the key to things being cleared out automatically as well, but as we've seen they weren't.

* [`result_persistent`](https://docs.celeryq.dev/en/stable/userguide/configuration.html#result-persistent) should default to `False` but I won't argue with being explicit about it. 🙂

**Review by ccostino — COMMENTED** [review]
Thanks, @terrazoon!  I noted a few things after looking into the settings some, just to double check on them.

I'd definitely like to double check the [`result_expires`](https://docs.celeryq.dev/en/stable/userguide/configuration.html#result-expires) setting in particular and make sure that's correct.  Thanks!

File: app/config.py (hunk 1 above)
Comment (terrazoon) [reply app/config.py]: You're right, I removed result_expires.

**Review by ccostino — APPROVED** [review]
Thanks, @terrazoon!

**Pull request merged**